eng/ci/public-build.yml

```diff
@@ -42,4 +42,16 @@ extends:
     - stage: Build
 
       jobs:
-      - template: /eng/ci/templates/jobs/build.yml@self
\ No newline at end of file
+      - template: /eng/ci/templates/jobs/build.yml@self
+    - stage: TestWindows
+      dependsOn: Build
+
+      jobs:
+      - template: /eng/ci/templates/jobs/run-emulated-tests-windows.yml@self
+
+    - stage: TestLinux
+      dependsOn:
+      - TestWindows
```

Comment:
We should run the windows and linux emulators tests in parallel instead of sequential

Reply:
Also should we run the emulators test as part of the official build as well?

Reply:
unfortunately the tests are executed against same set of resources. i dont have previous context but i guess it could bring race condition issues. its something can be tried later.
for official build i think its better to use real resource for internal e2e test for full verification. this is for public more 
![image](https://github.com/user-attachments/assets/709b77b3-79a1-47c2-a3fe-ea75d1b595a6)